src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
+
     public static void main(String[] args) {
+        String divider = "\t____________________________________________________________";
+        Scanner in = new Scanner(System.in);
+       // ArrayList<Tasks> taskList = new ArrayList<>();
+        String line;
         String logo = " ____        _        \n"
```

Comment:
1. Maybe the long print statements can be put into another class to look neater?